homeassistant/components/plugwise/climate.py

```diff
@@ -35,6 +33,7 @@
 
 HVAC_MODES_HEAT_ONLY = [HVAC_MODE_HEAT, HVAC_MODE_AUTO, HVAC_MODE_OFF]
 HVAC_MODES_HEAT_COOL = [HVAC_MODE_HEAT, HVAC_MODE_COOL, HVAC_MODE_AUTO, HVAC_MODE_OFF]
+THERMOSTAT_CLASSES = ["thermostat", "zone_thermostat", "thermostatic_radiator_valve"]
```

Comment:
Please copy from Plugwise-beta:
```
MASTER_HERMOSTATS = [
    "thermostat",
    "zone_thermometer",
    "zone_thermostat",
    "thermostatic_radiator_valve",
]
```
I can understand that you don't want to use MASTER, but there are 4 types that can represent a primary thermostat.

Reply:
This PR is not adding new functionalities and is limited to code quality changes.

Reply:
Ok, got it, sorry.

Reply:
No problem, made a note though, so it won't be forgotten 👍